### PR TITLE
Add GitHub actions for SDK and codegen CI testing

### PR DESCRIPTION
**Pull request by jasdel**
Adds GitHub Actions for Go 1.15, 1.16 SDK unit testing and Java codegen. Go tip is not included yet, relying on to rely on Travis CI for these until added to GitHub Actions.

### CODE REVIEW

**Comment by jasdel** [comment]
Looks like test are overall faster than Travis, also codegen java 11 failed on Travis. We should consider dropping all Travis tests except Go tip tests.

File: .github/workflows/codegen.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    - uses: actions/checkout@v2
+
+    - uses: actions/setup-java@v2
+      with:
+        distribution: 'zulu'
+        java-version: ${{ matrix.java-version }}
+
+    - uses: actions/setup-go@v2
+      with:
+        go-version: ${{ matrix.go-version }}
```
Comment (skmcgrail) [inline .github/workflows/codegen.yml:30]: This is much nicer then what I had to do with Travis CI recently.

File: .github/workflows/go.yml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+      matrix:
+        os: [windows-latest]
+        go-version: [1.16, 1.15]
+        #experimental: [false]
+        #include:
+        #  - go-version: tip
+        #    os: [windows-latest]
+        #    experimental: true
+    env:
+      EACHMODULE_SKIP: "internal\\repotools\\changes"
```
Comment (skmcgrail) [inline .github/workflows/go.yml:51]: We can probably remove this code now, we can do that in a separate PR.

**Review by skmcgrail — APPROVED** [review]
